chandra_aca/star_probs.py

```diff
@@ -329,6 +329,49 @@ def acq_success_prob(
     return probs[0] if is_scalar else probs
 
 
+def get_default_acq_prob_model_info(verbose=True):
+    """Get info about the default acquisition probability model.
+
+    Example::
+
+        >>> from chandra_aca import star_probs
+        >>> star_probs.get_default_acq_prob_model_info()
+        {'default_model': 'grid-*',
+         'call_args': {'file_path': 'chandra_models/aca_acq_prob',
+          'version': None,
+          'repo_path': 'None',
+          'require_latest_version': False,
+          'timeout': 5,
+          'read_func': '<function _read_grid_func_model at 0x11a443b50>',
+          'read_func_kwargs': {'model_name': None}},
+         'version': '3.48',
+         'commit': '68a58099a9b51bef52ef14fbd0f1971f950e6ba3',
+         'data_file_path': '/Users/aldcroft/ska/data/chandra_models/chandra_models/aca_acq_prob/grid-floor-2020-02.fits.gz',
```

Comment:
I note for me, for example, on OSX this data_file_path ends up showing the location of the temporary directory that was used.  A little weird but seems benign.